cst438/src/main/java/cst438/domain/CityRepository.java

```diff
@@ -3,9 +3,13 @@
 import java.util.List;
 
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.stereotype.Repository;
 
 @Repository
 public interface CityRepository extends JpaRepository<City, Long> {
 	List<City> findAllByOrderByNameAsc();
-}
+	
+	@Query("SELECT a FROM city a WHERE a.countryCode=?1 ORDER BY a.name ASC")
+	List<City> findAllByOrderByNameAsc(String countryCode);
```

Comment:
can we remove the query and change the function name to use the built in JPA function: `List<City> findAllByCountryCodeOrderByNameAsc(String countryCode);`

Reply:
+1 to this, but could just be a future refactor for the sake of time